rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -335,6 +412,8 @@ private void purgeRequests() {
     }
 
     synchronized void clean() {
+        logger.trace("clean - Cleaning expired requests with state: {}", state);
```

Comment:
we already print a WARN if `state != ExecState.RUNNING`, which should be rather a rare / exceptional case. I guess we could simply do `logger.trace("clean - Cleaning obsolete requests ...");` just above `this.purgeRequests();`, wdyt?